roles/solr9cloud/templates/solr.in.sh.j2

```diff
@@ -0,0 +1,39 @@
+# roles/solrcloud/templates/solr.in.sh.j2
+{{ ansible_managed | comment }}
+# Local customizations for Solr
+
+# Memory settings
```

Comment:
I wonder if we can just hardcode a bunch of these? Do we have significant differences for these variables in our cluster?